### PR TITLE
Forward resume options

### PR DESCRIPTION
**Pull request by dfunckt**
This was missed from #483 and user-set resume options were ignored.

Change-Type: patch

### CODE REVIEW

File: src/docker-utils.coffee [hunk 1]
```diff
@@ -52,9 +52,9 @@ getRepoAndTag = (image) ->
 			registry = ''
 		return { repo: "#{registry}#{imageName}", tag: tagName }
 
-applyDelta = (imgSrc, deltaUrl, { requestTimeout, applyTimeout, retryCount, retryInterval }, onProgress) ->
+applyDelta = (imgSrc, deltaUrl, requestTimeout, applyTimeout, resumeOpts, onProgress) ->
```
Comment (pcarranzav) [inline src/docker-utils.coffee]: I'd rather keep the function signature as `(imgSrc, deltaUrl, options, onProgress)`, i.e. making this `(imgSrc, deltaUrl, { requestTimeout, applyTimeout, resumOpts }, onProgress)`, what do you think?
Either that or keep the function definition as before and build `resumeOpts` inside this function.

Comment (dfunckt) [reply src/docker-utils.coffee]: Sounds good -- PR updated.

**Comment by dfunckt** [comment]
@pcarranzav -- it would be nice to have this with the next OS release.

**Comment by pcarranzav** [comment]
@dfunckt merging, will add to next release

**Comment by dfunckt** [comment]
Thanks 👍

**Pull request merged**